css/css-ui-3/text-overflow-015.html

```diff
@@ -0,0 +1,29 @@
+﻿<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: ellipsis after graphical transformations</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#ellipsing-details">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem">
+<meta name="assert" content="Ellipsing occurs after relative positioning and other graphical transformations">
```

Comment:
This is only testing relative positioning, so the assert needs to be that specific.

You also need a test for transforms.